components/nifosg/particle.hpp

```diff
@@ -135,6 +137,8 @@ namespace NifOsg
         GrowFadeAffector();
         GrowFadeAffector(const GrowFadeAffector& copy, const osg::CopyOp& copyop = osg::CopyOp::SHALLOW_COPY);
 
+        GrowFadeAffector& operator=(const GrowFadeAffector&) = default;
```

Comment:
This operator is used inside ctor like:
```c++
*this = copy;
```
This expression will call assignment operator for the base class which is probably not desired here. See [this](https://godbolt.org/z/jtNXyG) example. I suggest to mark assignment operator as deleted and change `*this = copy` to explicit members copy.

Reply:
Done.